src/components/Contact.js

```diff
@@ -1,13 +1,115 @@
 import React, { Component } from "react";
-import { Header, Form, Button, Responsive } from "semantic-ui-react";
+import axios from "axios";
+
+import { Header, Form, Button } from "semantic-ui-react";
+
+const PostMsg = ({ err }) =>
+  err ? (
+    <div>
+      <Header as="h3" textAlign="center" style={{ color: "red" }}>
+        Error while submitting your info. If possible, kindly send us the
+        screenshot of this page at bugs@iprhub.io...
+      </Header>
+    </div>
+  ) : (
+    <Header as="h3" textAlign="center" style={{ color: "green" }}>
+      Thank you, we will get back to you soon with the info provided...
+    </Header>
+  );
 
 export default class Contact extends Component {
-  state = {
-    fname: "",
-    lname: "",
-    email: "",
-    phoneno: "",
-    msg: ""
+  constructor(props) {
+    super(props);
+    this.state = {
+      // state = {
+      fName: "",
```

Comment:
use firstName instead of fName, follow the same descriptive convention for all the fields :)